actions/class.Runner.php

```diff
@@ -618,10 +621,34 @@ public function submitItem()
 
     /**
      * Moves the current position to the provided scoped reference: item, section, part
+     *
+     * This action is called when the user sends a test response, but not when
+     * the test starts.
      */
     public function move()
     {
         try {
+            $serviceContext = $this->getServiceContext();
+            $state = $serviceContext->getTestSession()->getState();
+
+            if ($state == AssessmentTestSessionState::SUSPENDED) {
+                $this->getLogger()->debug(
+                    sprintf(
+                        '%s: session %s is suspended, returning a PausedException response',
+                        self::class,
+                        $serviceContext->getTestSession()->getSessionId()
+                    )
+                );
+
+                $this->returnJson(
+                    $this->getErrorResponse(
+                        new QtiRunnerPausedException()
+                    )
+                );
+
+                return;
+            }
```

Comment:
**issue:** This breaks the pause feature when timer target is set to server!

This happens because the move action is stopped before having a chance to settle the timer. When the proctor pauses the test, only the state is updated, then the state is consolidated when the test taker tries to move. Since the proctor paused the test, the test runner has no way to close the timer since the move action is rejected before applying the rules...

**suggestion:** Remove this code. I don't think it is needed. The move action already handles it internally.

Reply:
Code was reverted as it is not needed anymore. Thanks! :)